src/Schema/ContextModule.ts

```diff
@@ -12,39 +13,53 @@ export enum ContextModule {
   artworkImage = "artworkImage",
   artworkSidebar = "artworkSidebar",
   auctionSidebar = "auctionSidebar",
+  auctionCard = "auctionCard",
   auctionRail = "auctionRail",
   auctionResults = "auctionResults",
   auctionsInfo = "auctionsInfo",
   bannerPopUp = "bannerPopUp",
   browseFair = "browseFair",
   categoryRail = "categoryRail",
   collectionDescription = "collectionDescription",
+  collectionRail = "collectionRail",
+  collectionCard = "collectionCard",
   consignSubmissionFlow = "consignSubmissionFlow",
   currentShowsRail = "currentShowsRail",
   fairInfo = "fairInfo",
   fairsHeader = "fairsHeader",
+  fairRail = "fairRail",
+  fairCard = "fairCard",
   featuredArtistsRail = "featuredArtistsRail",
+  featuredCollection = "featuredCollection",
```

Comment:
going with this name for the following module: 
<img width="301" alt="Screen Shot 2020-04-23 at 2 58 00 PM" src="https://user-images.githubusercontent.com/26879583/80138519-eac89000-8572-11ea-9b58-b7104ba80f67.png">

but open to other names. the [designs](https://www.figma.com/file/gzAi9BJi62AaJuhMRsgXMj/Home?node-id=2950%3A0) refer to 'promotions' and 'promo space' but "featuredCollection" covers most cases I can think of. perhaps "featuredCollectionCard" or "featuredCollectionRail" to match what we do for featured Artists?